README.md

```diff
@@ -45,7 +46,7 @@ This maven plugin allows to validate yaml and json files.
       </plugin>
 ```
 
-Validation failures make the build fail.
+Validation failures make the build fail but default. You can change this by setting `<failOnErrors>false</failOnErrors>` and error messages will be logged as warnings.
```

Comment:
At first sight when I see such a param as failOnErrors, I just expect errors to remain errors but just avoid failing te build. So, the severity should not change from error to warning.

Reply:
You are right, fixed